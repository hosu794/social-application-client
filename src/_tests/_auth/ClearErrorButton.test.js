const { shallow } = require("enzyme");

import React from "react";
const {
  default: ClearErrorButton,
} = require("../../_components/auth/ClearErrorButton");

let randomText = "Some text";

describe("Test for the ClearErrorButton", () => {
  let wrapper = shallow(<ClearErrorButton>{randomText}</ClearErrorButton>);

  test("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  test("should component have correctly text value", () => {
    let element = wrapper.find("div").text();

    expect(element).toEqual(randomText);
  });
});
