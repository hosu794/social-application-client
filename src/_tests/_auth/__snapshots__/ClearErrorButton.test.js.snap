// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the ClearErrorButton should render correctly 1`] = `ShallowWrapper {}`;
